src/Components/Filtering/Multiselect.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable no-unused-vars */
```

Comment:
Can you rename this file and MultiselectComponent.js to be clearer in relation to each other? Like maybe name this one MultiselectFilters and the other one MultiselectFilterOptions? Theses are pretty poor suggestions, but names that more clearly define their relationship would be helpful.